lib/onboarding.py

```diff
@@ -1,20 +1,15 @@
 import requests
 import sublime
 
-from ..lib import link_opener
+from ..lib import link_opener, languages, settings
 
-def start_onboarding():
-    """Attempts to open the live onboarding file. If fetching the file fails,
-    then a help dialog is shown instead.
-
-    This function returns whether or not the help prompt should be suppressed
-    in future startups. The help prompt should be suppressed if fetching the
-    live onboarding file succeeds or if the user clicked on the "No" button
-    in the help dialog.
+def start_onboarding(ext):
+    """Attempts to open the live onboarding file for a given language. 
+    If fetching the file fails, then a help dialog is shown instead.
     """
-
+    lang = languages.ext_to_lang(ext).lower()
     url = 'http://localhost:46624/clientapi/plugins/onboarding_file'
-    resp = requests.get(url, params={'editor': 'sublime3'})
+    resp = requests.get(url, params={'editor': 'sublime3', "language": lang})
 
     if resp.status_code != 200:
```

Comment:
Since this is Python specific, change this copy to:

```
Sublime Text is now integrated with Kite for Python

Kite provides ML-powered completions and instant documentation for Python to help keep you in the flow.

Would you like to learn how to use Kite for Python?
```